lib/background/modules/apps_background.dart

```diff
@@ -37,13 +37,13 @@ class AppsBackground implements BackgroundAppInstallCallbacks {
     );
   }
 
-  Future<void> onWatchConnected(PebbleDevice watch, bool unfaithful) async {
+  Future<bool> onWatchConnected(PebbleDevice watch, bool unfaithful) async {
     if (unfaithful) {
       Log.d('Clearing all apps and re-syncing');
-      await watchAppsSyncer.clearAllAppsFromWatchAndResync();
+      return await watchAppsSyncer.clearAllAppsFromWatchAndResync();
```

Comment:
Personally I avoid `return await delayed()` because it's mostly the same as `return delayed()` but shorter.
Sometimes there's a difference though: https://stackoverflow.com/a/42750371/1274643
In this case, I'd remove `await` and let downstream methods await returned Futures.